tms_objects2.erb

```diff
@@ -334,6 +334,17 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
               <lido:term><%= o["Period"] %></lido:term>
             </lido:periodName> 
         <% end %>
+            <lido:eventMaterialsTech> 
+              <lido:displayMaterial><%= MediumPS %></lido:displayMaterial> 
+              <lido:materialsTech> 
+        <% AATMediumTerms.each do |m| %> 
+                <lido:termMaterialTech> 
+                  <lido:conceptID lido:type = "<%= m["colThesxrefType"] %>" lido:source = "<%= m["UCTS"] %>"><%= m["FinalST"] %></lido:conceptID> 
```

Comment:
m["FinalST"] should be m["SourceTermID"]
(the value shouldn't start with "aat", in coboat there's a firstpass configuration which renames this field making it confusing here)